### PR TITLE
Updated and Created Readme for desired Topics

### PR DESCRIPTION
**Pull request by Ghat0tkach**
## Fixes Issue

Updated README with Roadmap links
Created Blockchain Roadmap readme

### CODE REVIEW

**Review by siddhant-khisty — APPROVED** [review]
LGTM.
cc @kaiwalyakoparkar

File: Mobile-Development/readme.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- [Activities Lifecycle And Cycle](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-1-get-started/lesson-2-activities-and-intents/2-2-c-activity-lifecycle-and-state/2-2-c-activity-lifecycle-and-state.html)
+
+
+
+## Networking And API's
+- [Notes-Slides](https://piazza.com/class_profile/get_resource/ktlu0ly5db84bb/kvj97fdgts92ju)
+- [Networking And API's](https://youtu.be/k2N3EoZI3eU)
+- [Retrofit](https://square.github.io/retrofit/)
+
+
```
Comment (siddhant-khisty) [inline Mobile-Development/readme.md:45]: Too much whitespace. Please leave only one line between topics

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

File: Mobile-Development/readme.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- [Developer Docs](https://developer.android.com/guide/topics/ui/ui-events)
+- [Event Handlers](https://youtu.be/LV_5lOvYAn8)
+
+## Views
+- [Notes-Slides](https://piazza.com/class_profile/get_resource/ktlu0ly5db84bb/kuz3vxzyo744i8)
+- [Recycler View](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-2-user-experience/lesson-4-user-interaction/4-5-c-recyclerview/4-5-c-recyclerview.html)
+- [Text And Scroll Views](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-1-get-started/lesson-1-build-your-first-app/1-3-c-text-and-scrolling-views/1-3-c-text-and-scrolling-views.html)
+- [Button And Clickable Images](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-2-user-experience/lesson-4-user-interaction/4-1-c-buttons-and-clickable-images/4-1-c-buttons-and-clickable-images.html)
+
+
```
Comment (kaiwalyakoparkar) [inline Mobile-Development/readme.md:30]: Remove extra line

File: Mobile-Development/readme.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+## Activities & Intents
+- [Notes-Slides](https://piazza.com/class_profile/get_resource/ktlu0ly5db84bb/kuz3w4v5gpn4zh)
+- [Implicit Intent](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-1-get-started/lesson-2-activities-and-intents/2-3-c-implicit-intents/2-3-c-implicit-intents.html)
+- [Intents And Starting Activities](https://youtu.be/1xj9G2FvLeE)
+- [Activites And Intent](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-1-get-started/lesson-2-activities-and-intents/2-1-c-activities-and-intents/2-1-c-activities-and-intents.html)
+- [Activities Lifecycle And Cycle](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-1-get-started/lesson-2-activities-and-intents/2-2-c-activity-lifecycle-and-state/2-2-c-activity-lifecycle-and-state.html)
+
+
+
```
Comment (kaiwalyakoparkar) [inline Mobile-Development/readme.md:39]: Remove above extra 2 line

File: Mobile-Development/readme.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## Networking And API's
+- [Notes-Slides](https://piazza.com/class_profile/get_resource/ktlu0ly5db84bb/kvj97fdgts92ju)
+- [Networking And API's](https://youtu.be/k2N3EoZI3eU)
+- [Retrofit](https://square.github.io/retrofit/)
+
+
+
+
+
```
Comment (kaiwalyakoparkar) [inline Mobile-Development/readme.md:48]: Remove the additional blank lines

File: Mobile-Development/readme.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+## Fragments
+- [Fragments Lecture](https://youtu.be/JuBpWzEoDw4)
+- [Fragments Implimentation](https://youtu.be/Btli00YA1eI)
+
+
+
```
Comment (kaiwalyakoparkar) [inline Mobile-Development/readme.md:54]: Too many blank lines

File: Mobile-Development/readme.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+## Application Architecture
+- [Article on App Architecture](https://developer.android.com/topic/architecture)
+- [Android's Guide To App Architecture](https://youtu.be/4L3-DW-z7iI)
+
+
+
```
Comment (kaiwalyakoparkar) [inline Mobile-Development/readme.md:60]: Again same, too many blank lines. Remove them

File: Mobile-Development/readme.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+- [Introduction To Kotlin](https://youtu.be/X1RVYt2QKQE)
+- [Kotlin In 12 Minutes](https://youtu.be/iYrgWO2oibY)
+- [Kotlin Course](https://developer.android.com/courses/pathways/android-basics-kotlin-one)
+
+## Layouts And UI
+- [Notes-Slides](https://piazza.com/class_profile/get_resource/ktlu0ly5db84bb/kubrwffwj8459a)
+- [Layouts And UI](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/unit-1-get-started/lesson-1-build-your-first-app/1-2-c-layouts-and-resources-for-the-ui/1-2-c-layouts-and-resources-for-the-ui.html)
+- [Layouts,UI & Responding To User Input](https://youtu.be/o1SZJNYJ7vg)
+
+
```
Comment (kaiwalyakoparkar) [inline Mobile-Development/readme.md:18]: Too many blank lines, kindly remove them

File: Blockchain/readme.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+- [Oracles](https://chain.link/education/blockchain-oracles)
+- [What Are NFTs](https://www.forbes.com/advisor/investing/cryptocurrency/nft-non-fungible-token/)
+- [What iS A DAO](https://www.forbes.com/sites/cathyhackl/2021/06/01/what-are-daos-and-why-you-should-pay-attention/?sh=343b04067305)
+- [Intro To DAPPS](https://ethereum.org/en/developers/docs/dapps/)
+
+### Choose A Blockchain
+- [Ethereum](https://ethereum.org/)
+- [Solana](https://solana.com/)
+
+
```
Comment (kaiwalyakoparkar) [inline Blockchain/readme.md:24]: too many blank lines, please remove them

File: Blockchain/readme.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
+# Blockchain
+
+## Introduction
+- [What is Web3](https://www.freecodecamp.org/news/what-is-web3/)
+
+
```
Comment (kaiwalyakoparkar) [inline Blockchain/readme.md:6]: Extra blank line please remove it

File: README.md [hunk 10]
```diff
@@ -5,9 +5,20 @@ Are you prepared to embark on your quest to receive mentoring and land a job? Yo
...
+<p  align="center"><a href=""><b>DevOps</b></a></p>
+<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/DevRel#readme"><b>DevRel</b></a></p>
+<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Frontend-Development#readme"><b>Frontend-Development</b></a></p>
+<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Fullstack-Development#readme"><b>Fullstack-Development</b></a></p>
+
+<p  align="center"><a href=""><b>Android-Development</b></a></p>
+<p align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Open-Source#readme"><b>Blockchain</b></a></p>
+
+
+
```
Comment (kaiwalyakoparkar) [inline README.md:21]: Additional blank lines, please remove them

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]
Please add linters where needed

**Review by siddhant-khisty — APPROVED** [review]
Looks good to me. Thanks for your contribution

File: Blockchain/readme.md [hunk 11]
```diff
@@ -77,18 +67,15 @@
...
 -  [Scaffold-Ethereum](https://docs.scaffoldeth.io/scaffold-eth/)
-  
-### SDKs
+  ### SDKs
 - [Third Web](https://portal.thirdweb.com/)
 - [Moralis](https://docs.moralis.io/moralis-dapp/getting-started)
-
 ## Project Ideas
 - Article Format
                              <P>[Resource 1](https://dev.to/edge-and-node/the-complete-guide-to-full-stack-web3-development-4g74)</P>
```
Comment (kaiwalyakoparkar) [inline Blockchain/readme.md]: change the `<p>` tag to markdown format

File: Blockchain/readme.md [hunk 12]
```diff
@@ -77,18 +67,15 @@
 - Testing Tools
                              <P>[1) Waffle](https://ethereum-waffle.readthedocs.io/en/latest/)</P>
```
Comment (kaiwalyakoparkar) [inline Blockchain/readme.md:76]: change the `<p>` tag to markdown format

File: Blockchain/readme.md [hunk 13]
```diff
@@ -40,12 +35,10 @@
                             <P>[1)Alchemy](https://www.alchemy.com/)</P>
```
Comment (kaiwalyakoparkar) [inline Blockchain/readme.md:38]: change the `<p>` tag to markdown format, Alignment issue as well

File: Blockchain/readme.md [hunk 14]
```diff
@@ -16,17 +14,14 @@
...
 - [Ethereum](https://ethereum.org/)
 - [Solana](https://solana.com/)
-
-
 ### Ethereum Concepts
 - [Accounts](https://ethereum.org/en/developers/docs/accounts/)
 - [Transactions](https://ethereum.org/en/developers/docs/transactions)
 - Consensus :
-                            <P>[1) Proof Of Work](https://www.investopedia.com/terms/p/proof-work.asp)</P>
+                             <P>[1) Proof Of Work](https://www.investopedia.com/terms/p/proof-work.asp)</P>
```
Comment (kaiwalyakoparkar) [inline Blockchain/readme.md:27]: change the `<p>` tag to markdown format

**Comment by Ghat0tkach** [comment]
I  did the desired changes and apologize for the inconvenience

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

**Pull request merged**